cmd/utils/config.go

```diff
@@ -529,7 +529,7 @@ func (kCfg *KlayConfig) SetKlayConfig(ctx *cli.Context, stack *node.Node) {
 		cfg.WorkerDisable = true
 	}
 	cfg.RocksDBConfig.CacheSize = ctx.Uint64(RocksDBCacheSizeFlag.Name)
-	cfg.RocksDBConfig.DumpMallocStat = ctx.Bool(RocksDBDumpMallocStatFlag.Name)
+	cfg.RocksDBConfig.DumpMallocStat = ctx.Bool(RocksDBDumpMemoryStatFlag.Name)
```

Comment:
Don't we need to change the name of this config?
```suggestion
	cfg.RocksDBConfig.DumpMemoryStatFlag = ctx.Bool(RocksDBDumpMemoryStatFlag.Name)
```

Reply:
Talked to jk, turns out `malloc` is more apt. I'll change the flag name instead.